src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -417,9 +417,8 @@ private static bool RemovalMayIntroduceInterpolationAmbiguity(ParenthesizedExpre
                         node.Expression.Kind() == parentBinaryExpression.Kind() &&
```

Comment:
```suggestion
                        node.Expression.IsKind(parentBinaryExpression.Kind(), out BinaryExpressionSyntax binaryExpression) &&
```

Reply:
thanks!  i'd forgotten about that  guy :)